src/components/CCard.vue

```diff
@@ -142,6 +142,11 @@ const props = withDefaults(
      */
     subtitle?: string;
 
+    /**
+     * The variant how this `CCard` is used.
+     */
+    usageVariant?: "default" | "bottom-sheet";
+
```

Comment:
Why not just `variant`?

Reply:
Because the property `variant` already exists for vuetify `v-card` with the options `outlined, tonal, etc.`

Reply:
I don't think `usage` helps a lot to differentiate those. We should think of another name IMO.

I was thinking `type` maybe?